packages/components/src/FilterSelect/FilterSelect.spec.tsx

```diff
@@ -185,7 +185,7 @@ describe("FilterSelect generic", () => {
         {({ items }): JSX.Element[] =>
           items.map(item =>
             item.type === "item" ? (
-              <li key={item.key}>{item.value.isRubberDuck}</li>
+              <li key={item.key}>{item.value?.isRubberDuck}</li>
```

Comment:
How did this break by updating the date-picker? 😅

Reply:
I wrote the reason in the commit message, it's a weird one.

I'm going to ask @gyfchong to take a look and decide if this is the best solution or not

Reply:
If they want to allow `null` then we'll have to check for it. Sounds like good practice to always check for null anyway.